### PR TITLE
Merge OpenApiXmlComment generator to main

### PR DESCRIPTION
**Pull request by captainsafia**
This has been sitting in a branch for a while but merging it to main so we can churn out a NuGet package and start to iterate on this prototype more seriously.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+        "9.0.0-rc.2.24473.5",
+        "6.0.0"
       ],
       "aspnetcore": [
-        "6.0.3"
+        "9.0.0-rc.2.24474.3",
+        "6.0.0",
+        "3.1.23"
       ],
       "aspnetcore/x64": [
```
Comment (wtgodbe) [inline global.json:17]: Might as well remove this whole block while you're here (x64 & x86)

File: global.json [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
       "dotnet": [
-        "6.0.3"
+        "9.0.0-rc.2.24473.5",
+        "6.0.0"
       ],
       "aspnetcore": [
-        "6.0.3"
+        "9.0.0-rc.2.24474.3",
+        "6.0.0",
+        "3.1.23"
```
Comment (wtgodbe) [inline global.json:15]: Remove

**Review by wtgodbe — APPROVED** [review]
Didn't look at the content of the addition, but the changes generally look good

File: global.json [hunk 3]
```diff
@@ -13,12 +13,6 @@
         "9.0.0-rc.2.24474.3",
         "6.0.0",
         "3.1.23"
-      ],
```
Comment (wtgodbe) [inline global.json:13]: You might actually need to keep this, I think the format was different in 3.1

File: global.json (hunk 2 above)
Comment (captainsafia) [reply global.json:15]: Per offline conversation, there are still projects that target `netcoreapp3.1` in this repo. We can probably think about deleting them separately.

File: global.json (hunk 3 above)
Comment (captainsafia) [reply global.json:13]: I suppose CI will let us know soon enough?

**Pull request merged**